src/test/RwaSpell.t.sol

```diff
@@ -778,12 +755,12 @@ contract DssSpellTest is DSTest, DSMath {
     }
 
     function testSpellIsCast_RWA001_INTEGRATION_TELL_CURE_GOOD() public {
-        vote();
+        vote(address(spell));
         scheduleWaitAndCast();
         assertTrue(spell.done());
 
         tellSpell = new TellSpell();
-        voteTemp(address(tellSpell));
+        vote(address(tellSpell));
 
         tellSpell.schedule();
 
```

Comment:
We should do the same with `spellWaitAndCast`, just pass the `spell` as an argument.

Reply:
I was thinking the same thing, but perhaps in another PR.